distribution/client/test/update.test.js

```diff
@@ -26,6 +25,14 @@ describe('The update module', () => {
     app = feathers();
     update = new Update(app);
     update.healthChecker = new HealthChecker('http://127.0.0.1:8080/', { delay: 50, retry: 5});
```

Comment:
I think this line could be simply a `new Object()`, but well, that shouldn't be biggy different.